markdown-mode.el

```diff
@@ -9730,6 +9730,7 @@ rows and columns and the column alignment."
                            ;; not paragraph-ending suffixes:
                            ".*  $" ; line ending in two spaces
                            "^#+"
+                           "^[-=]+"
```

Comment:
This seems to be breaking paragraph fill of list elements started by a `-` at the beginning of the line, is this behaviour intended?

Reply:
Thanks for comment. I didn't consider about list syntax. I'll fix it later.

Reply:
Thanks a lot for answering so quickly!